src/app/pages/AccountPage/index.tsx

```diff
@@ -116,8 +117,17 @@ export function AccountPage(props: Props) {
       )}
       {account.accountError && (
         <p>
-          {t('account.loadingError', "Couldn't load account. Information may be missing or out of date.")}{' '}
-          {account.accountError}
+          {t('account.loadingError', "Couldn't load account.")}{' '}
+          <ErrorFormatter code={account.accountError.code} message={account.accountError.message} />
```

Comment:
> @tjanez Looking at the screenshots, I think it would make sense to make these warnings about unreachable API backends stand out a bit more.

choose:
| error | warning |
| --- | --- |
|  ![localhost_3000_account_oasis1qqsehts9fvmhujcsqtnx5c0wg6zcuxt9dulsatty](https://user-images.githubusercontent.com/3758846/176283141-ad2f765c-0891-4934-8c55-425e613f9ad5.png) | ![localhost_3000_account_oasis1qqsehts9fvmhujcsqtnx5c0wg6zcuxt9dulsatty (1)](https://user-images.githubusercontent.com/3758846/176283148-dc01007b-cc3c-4742-b112-188e6f4d5a4c.png)  |